<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QToolButton{
		background-color: #44c767;
		border-radius: 4px;
		color: #ffffff;
		font-family: 'Roboto';
		margin-left:0.5em;
		font-size: 17px;
	}
QToolButton:hover{
			background-color:#8fe860;
	}

QToolBar{
	border-color:#8fe860;
}*/
/*QTextEdit{
		border-radius: 3px;
		border: 2px solid #00796b;
	}
QToolButton{
		background-color:#44c767;
		color:#ffffff;
		border-radius: 16px;
		margin-left : 1em;
		font-family:Arial;
		font-size:15px;
}
	
QToolButton:hover{
			background-color:#8fe860;
	}
*/
</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="btnNuevo"/>
    <addaction name="btnAbrir"/>
    <addaction name="btnGuardar"/>
    <addaction name="separator"/>
    <addaction name="btnSalir"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="btnInformacion"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="btnFijar"/>
    <addaction name="btnLexer"/>
    <addaction name="btnContinuar"/>
    <addaction name="btnPausar"/>
    <addaction name="btnLimpiarTerminal"/>
    <addaction name="btnLimpiarVariable"/>
    <addaction name="separator"/>
    <addaction name="btnIniciar"/>
    <addaction name="btnCancelar"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="btnBreakpoints"/>
    <addaction name="btnTiempos"/>
    <addaction name="btnArbol"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="Barra">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="btnAbrir">
   <property name="icon">
    <iconset>
     <normalon>:/Open/Enter.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="btnGuardar">
   <property name="icon">
    <iconset>
     <normalon>:/Save/Download.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="btnInformacion">
   <property name="icon">
    <iconset resource="../../../../interprete-analisis-master/Imagenes/resources.qrc">
     <normaloff>:/Info/Info.png</normaloff>:/Info/Info.png</iconset>
   </property>
   <property name="text">
    <string>Informaci√≥n</string>
   </property>
  </action>
  <action name="btnFijar">
   <property name="icon">
    <iconset>
     <normalon>:/Line/Pin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Fijar</string>
   </property>
  </action>
  <action name="btnLexer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../interprete-analisis-master/Imagenes/resources.qrc">
     <normaloff>:/Lexer/Lightning.png</normaloff>:/Lexer/Lightning.png</iconset>
   </property>
   <property name="text">
    <string>L&amp;exer</string>
   </property>
  </action>
  <action name="btnSalir">
   <property name="icon">
    <iconset>
     <normalon>:/Close/Cancel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="btnContinuar">
   <property name="icon">
    <iconset>
     <normalon>:/Continue/FlagBlue.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </action>
  <action name="btnPausar">
   <property name="icon">
    <iconset>
     <normalon>:/Stop/FlagYellow.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="btnNuevo">
   <property name="icon">
    <iconset>
     <normalon>:/Add/Add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="btnIniciar">
   <property name="icon">
    <iconset>
     <normalon>:/Run/FlagGreen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="btnCancelar">
   <property name="icon">
    <iconset>
     <normalon>:/Cancel/FlagRed.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ca&amp;ncel</string>
   </property>
  </action>
  <action name="btnLimpiarVariable">
   <property name="icon">
    <iconset>
     <normalon>:/Clear/Bin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Limpiar Variable</string>
   </property>
  </action>
  <action name="btnLimpiarTerminal">
   <property name="icon">
    <iconset>
     <normalon>:/Clear/Bin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Limpirar Terminal</string>
   </property>
  </action>
  <action name="btnBreakpoints">
   <property name="text">
    <string>Breakpoints</string>
   </property>
  </action>
  <action name="btnArbol">
   <property name="text">
    <string>Tree</string>
   </property>
  </action>
  <action name="btnTiempos">
   <property name="text">
    <string>Times</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../interprete-analisis-master/Imagenes/resources.qrc"/>
 </resources>
 <connections/>
</ui>
